### PR TITLE
RethinkDB 2.2.3

### PR DESCRIPTION
**Pull request by dalanmiller**

### CODE REVIEW

File: library/rethinkdb [hunk 1]
```diff
@@ -1,31 +1,32 @@
...
+2.1.4: git://github.com/rethinkdb/rethinkdb-dockerfiles@9244272dd4cdaea7b61b2321f827b2a70c4cfa9f jessie/2.1.4
+2.1.5: git://github.com/rethinkdb/rethinkdb-dockerfiles@9244272dd4cdaea7b61b2321f827b2a70c4cfa9f jessie/2.1.5
+2.1: git://github.com/rethinkdb/rethinkdb-dockerfiles@9244272dd4cdaea7b61b2321f827b2a70c4cfa9f jessie/2.1.5
+2.2.0: git://github.com/rethinkdb/rethinkdb-dockerfiles@9244272dd4cdaea7b61b2321f827b2a70c4cfa9f jessie/2.2.0
+2.2.1: git://github.com/rethinkdb/rethinkdb-dockerfiles@9244272dd4cdaea7b61b2321f827b2a70c4cfa9f jessie/2.2.1
+2.2.2: git://github.com/rethinkdb/rethinkdb-dockerfiles@9244272dd4cdaea7b61b2321f827b2a70c4cfa9f jessie/2.2.2
+2.2.3: git://github.com/rethinkdb/rethinkdb-dockerfiles@9244272dd4cdaea7b61b2321f827b2a70c4cfa9f jessie/2.2.2
+2.2: git://github.com/rethinkdb/rethinkdb-dockerfiles@9244272dd4cdaea7b61b2321f827b2a70c4cfa9f jessie/2.2.1
+2: git://github.com/rethinkdb/rethinkdb-dockerfiles@9244272dd4cdaea7b61b2321f827b2a70c4cfa9f jessie/2.2.1
+latest: git://github.com/rethinkdb/rethinkdb-dockerfiles@9244272dd4cdaea7b61b2321f827b2a70c4cfa9f jessie/2.2.2
```
Comment (yosifkit) [inline library/rethinkdb]: Hmm, we missed this last time too (:disappointed: in myself).  These last 4 lines should have the folder at the end bumped to `2.2.3`.

**Comment by dalanmiller** [comment]
Good catch @yosifkit, thank you.

**Comment by yosifkit** [comment]
This looks fine to me!
Changeset: https://github.com/rethinkdb/rethinkdb-dockerfiles/compare/7a7801cc5b82e9d...9244272dd4cd

(starting local build test)

*Note: If you guys want to do less versions here, that is fine by us. This list in the `library/` controls the "supported tags" list on the Docker Hub as well as which ones get rebuilt (which usually happens at least once a month when debian images get updated); any other tags stay available on the Docker Hub, but are no longer "supported".  Most upstreams just list the latest version of each major release, and align that to which ones get security fixes.*

**Comment by yosifkit** [comment]
LGTM, Build test of #1331; 3250cf8 (`rethinkdb`):

```console
$ bashbrew build "rethinkdb"
Cloning rethinkdb (git://github.com/rethinkdb/rethinkdb-dockerfiles) ...
Processing rethinkdb:1.15.1 ...
Processing rethinkdb:1.15.2 ...
Processing rethinkdb:1.15.3 ...
Processing rethinkdb:1.15 ...
Processing rethinkdb:1.16.0 ...
Processing rethinkdb:1.16.1 ...
Processing rethinkdb:1.16.2 ...
Processing rethinkdb:1.16.3 ...
Processing rethinkdb:1.16 ...
Processing rethinkdb:1 ...
Processing rethinkdb:2.0.0 ...
Processing rethinkdb:2.0.1 ...
Processing rethinkdb:2.0.2 ...
Processing rethinkdb:2.0.3 ...
Processing rethinkdb:2.0.4 ...
Processing rethinkdb:2.0 ...
Processing rethinkdb:2.1.0 ...
Processing rethinkdb:2.1.1 ...
Processing rethinkdb:2.1.2 ...
Processing rethinkdb:2.1.3 ...
Processing rethinkdb:2.1.4 ...
Processing rethinkdb:2.1.5 ...
Processing rethinkdb:2.1 ...
Processing rethinkdb:2.2.0 ...
Processing rethinkdb:2.2.1 ...
Processing rethinkdb:2.2.2 ...
Processing rethinkdb:2.2.3 ...
Processing rethinkdb:2.2 ...
Processing rethinkdb:2 ...
Processing rethinkdb:latest ...
$ bashbrew list --uniq "$url" | xargs test/run.sh
testing rethinkdb:1.15.1
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed
testing rethinkdb:1.15.2
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed
testing rethinkdb:1.15.3
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed
testing rethinkdb:1.16.0
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed
testing rethinkdb:1.16.1
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed
testing rethinkdb:1.16.2
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed
testing rethinkdb:1.16.3
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed
testing rethinkdb:2.0.0
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed
testing rethinkdb:2.0.1
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed
testing rethinkdb:2.0.2
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed
testing rethinkdb:2.0.3
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed
testing rethinkdb:2.0.4
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed
testing rethinkdb:2.1.0
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed
testing rethinkdb:2.1.1
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed
testing rethinkdb:2.1.2
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed
testing rethinkdb:2.1.3
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed
testing rethinkdb:2.1.4
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed
testing rethinkdb:2.1.5
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed
testing rethinkdb:2.2.0
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed
testing rethinkdb:2.2.1
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed
testing rethinkdb:2.2.2
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed
testing rethinkdb:2.2.3
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed
```

**Pull request merged**